src/web/mjs/engine/BookmarkManager.mjs

```diff
@@ -113,7 +115,6 @@ export default class BookmarkManager extends EventTarget {
      * Callback will be provided with an error (or null if no error).
      */
     loadProfile( profile, callback ) {
-        //HYPOFLEX
```

Comment:
removed the name tags, these were mainly used to quick search without knowning the exact function name.